db/models.py

```diff
@@ -14,3 +14,43 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="Movies")
+    genres = models.ManyToManyField(Genre, related_name="Movies")
```

Comment:
The `related_name` for the `ManyToManyField` in the `Movie` model should be in lowercase and plural form according to Django conventions. Consider changing `related_name="Movies"` to `related_name="movies"`.